src/main/resources/anniv5Resources/localization/eng/thieverypack/Cardstrings.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Kind of surprised there isn't a gold stealing card in this.

I like the flavor but not that enemies fulfill the conditions so some of these cards are hard to justify